lib/src/main/java/com/orange/ods/compose/component/banner/OdsBanner.kt

```diff
@@ -115,18 +118,18 @@ fun OdsBanner(
             }
         }
         if (!isSingleLineBanner && buttonCount > 0) {
-            Row(
+            FlowRow(
                 modifier = Modifier
-                    .padding(bottom = dimensionResource(id = R.dimen.spacing_xs))
                     .padding(horizontal = dimensionResource(id = R.dimen.spacing_m))
                     .align(Alignment.End),
-                horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.spacing_s))
+                horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.spacing_s), Alignment.End),
+                verticalArrangement = Arrangement.spacedBy((-6).dp)
             ) {
                 firstButton?.Content()
                 secondButton?.Content()
             }
         }
-        OdsDivider()
+        OdsDivider(modifier = Modifier.padding(top = dimensionResource(id = R.dimen.spacing_xs)))
```

Comment:
Moving the padding to the top of the divider also adds a space when there is no button (although it's hard to see it because it's only `4.dp`). Moving it back to the `FlowRow` looks good to me, but maybe there is an issue I did not see.